LoopKitUI/Extensions/View+KeyboardAware.swift

```diff
@@ -23,6 +23,8 @@ extension View {
 public struct KeyboardAware: ViewModifier {
     @State var keyboardHeight: CGFloat = 0
 
+    public init() { } // To enable initialization in other modules
```

Comment:
You can use the public `keyboardAware` method above this struct rather than exposing this initializer

Reply:
(In hindsight I probably should've made the whole KeyboardAware struct fileprivate to start with, feel free to make that change too)

Reply:
Sure, happy to do that